web-common/src/features/sources/modal/AddDataForm.svelte

```diff
@@ -104,6 +129,16 @@
   // Emit the submitting state to the parent
   $: dispatch("submitting", { submitting });
 
+  // Generate YAML preview from form state
+  $: yamlPreview = (() => {
+    const values = useDsn ? $dsnForm : $paramsForm;
+    return compileConnectorYAML(connector, values);
+  })();
+
+  $: showConnectorPreview = !(
+    isClickHouse && deploymentType === "rill-managed"
+  );
```

Comment:
We should still show the connector preview in this case. It'd help the user understand what file will be created.